utoipa/src/openapi/schema.rs

```diff
@@ -284,6 +284,10 @@ pub struct Discriminator {
     /// Defines a discriminator property name which must be found within all composite
     /// objects.
     pub property_name: String,
+
+    // An object to hold mappings between payload values and schema names or references.
```

Comment:
Can you change this to triple slash and expand the comment so it is more suitable for public documentation?
It needs to state that it can only be populated manually - i.e. there is no macro support, and there is no validation of the mapping.

Reply:
I edited the comment and added a test here: [c2834c5](https://github.com/juhaku/utoipa/pull/752/commits/c2834c575716219ba4226fc3ac6ed0c1ae0bd01e)